sia-react-native/Backend.js

```diff
@@ -45,6 +46,26 @@ class Backend extends React.Component {
     });
   }
 
+  formatDate(dateString) {
+  const parsed = moment(new Date(dateString));
+
+  if (!parsed.isValid()) {
+    return dateString;
+  }
+
+  return parsed.format('D MMM YYYY');
+}
+
+formatDateTime(dateString) {
+  const parsed = moment(new Date(dateString));
+
+  if (!parsed.isValid()) {
+    return dateString;
+  }
+
+  return parsed.format('H:mm A on D MMM YYYY');
+}
+
```

Comment:
Consider moving these outside of the Backend component since they aren't used in the backend.